pkg/grub/rootfs.cfg

```diff
@@ -259,6 +259,9 @@ function set_arm64_baremetal {
       smbios -t 1 -s 5 --set smb_product
       if [ "$smb_product" = "rpi" ]; then
          set_to_existing_file devicetree /boot/dtb/broadcom/bcm2711-rpi-4-b.dtb
+      elif [ "$smb_product" = "Raspberry Pi 5 Model B Rev 1.0" ]; then
+         #set_to_existing_file devicetree /boot/dtb/broadcom/bcm2712-rpi-5-b.dtb
```

Comment:
Don't we need the DTB file in this case?

Reply:
Initially, I used this, but this caused the mac address to be random on each boot. Only, when using the device tree provided by the firmware (which is pretty much the same as the one in the rootfs), we get the correct mac address.

Reply:
But I don't understand how we set the DTB in this case. 
And if we don't need it, we should remove the commented line...
Also, hoping @rene is taking a look here.

Reply:
If we set the device tree here, the kernel will use this one. Otherwise, it will use the one provided by the firmware/boot process (which is already loaded in the correct address). I left this as a comment, in case we find a different solution for the mac address in the future and prefer to use the kernel device tree.

Reply:
@OhmSpectator, the first dtb provided in the EFI partition (that comes from u-boot), will be dynamic updated by the firmware with relevant information, like the MAC address of the embedded NIC, so we are good to use this one instead of the provided by kernel as long as we don't put specific stuff in the kernel's dtb, which is not our case for RPi5. @dpoulios, please remove the commented line.

I think we can also do the same with RPi 4b, IIRC we just add the node for the TPM, which can be converted into an overlay... but it's okay to let this to a future PR....